include/vctr/Containers/VctrBase.h

```diff
@@ -366,6 +366,24 @@ class VctrBase : private Config,
     /** Subtracts a constant from this in place */
     void operator-= (value_type c);
 
+    //==============================================================================
+    // Math reduction operations
+    //==============================================================================
+    /** Returns the minimal value of all elements */
+    ElementType min() const requires std::totally_ordered<ElementType>;
+
+    /** Returns the minimal absolute value of all elements */
+    ElementType minAbs() const requires std::totally_ordered<ElementType>;
+
+    /** Returns the maximum value of all elements */
+    ElementType max() const requires std::totally_ordered<ElementType>;
+
+    /** Returns the maximum absolute value of all elements */
+    ElementType maxAbs() const requires std::totally_ordered<ElementType>;
+
+    /** Returns the sum of all elements */
+    ElementType sum() const requires has::operatorPlusEquals<ElementType>;
```

Comment:
I suggest (for the future) to also add the sum-of-squares, squared sum, and sum of the absolute value because I feel they are frequently needed. I'll check if there is an issue for that...

Reply:
Ah, I had even mentioned sum-of-squares in the issue description! Never mind, let's make a separate issue for that.

Reply:
Good point, I think they are not on our feature backlog now, so feel free to add an issue

Reply:
very interesting concept since you could also implement i.e. your own struct with overloaded += operator that does anything under the hood! nice choice.

Reply:
Thanks! I try to go routes like that wherever possible in the implementation